difftree/internal/radixmerkle/noder.go

```diff
@@ -0,0 +1,20 @@
+package merkletrie
+
+// The Noder interface is implemented by the elements of a Merkle Trie.
+type Noder interface {
```

Comment:
This should probably be in `utils/iterators`, but maybe we can wait until we want to actually use it for other algorithms (e.g. commit walk, tree walk).